BroadcastUploadExtension/Common.xcconfig

```diff
@@ -1,17 +1,8 @@
 // 
 // Copyright 2023 Vector Creations Ltd
```

Comment:
Are we able to set all of these to `// Copyright 2017-2024 New Vector Ltd`? or do they have to remain with the initial value that was used?

Reply:
I can definitely change them, I am just not sure if we should update this or not